web/e2e/page-objects/flows/manageFlows.ts

```diff
@@ -233,7 +233,7 @@ export class ManageFlows extends AppPage {
    * @param option = ["Previous page"/"Next page"]
    */
   async clickPageNavigation(option: string) {
-    return await element(by.css(`button[ng-reflect-message="${option}"]`)).click();
+    return await element(by.css(`button[aria-label="${option}"]`)).click();
```

Comment:
isn't the same thing? it goes to the same selector, it doesn't need to be changed

Reply:
I didn't see the attribute ng-reflect-message.

Reply:
Was there before. Not anymore. Could have used label or the class. Lets chk with UI team once for best practice. If class name is something that will stay longer for the element(Apparently label name has changed or been removed) then lets use class by.css{`button.mat-paginator-navigation-${option}`} where option is previous or next